src/LightGraphs.jl

```diff
@@ -259,6 +259,7 @@ include("vertexcover/degree_vertex_cover.jl")
 include("vertexcover/random_vertex_cover.jl")
 include("Experimental/Experimental.jl")
 include("Parallel/Parallel.jl")
+include("transitivity/transitivity.jl")
```

Comment:
If we have an extra file for transitivity, maybe we should also move `transitive_closure` and `transitive_reduction` for directed graphs to this file.

Reply:
I just did not know where to put it, so I created a new file. If you have an idea where it should go, I definitely can move it here.
Otherwise, yes, I could do the reverse and bring DiGraph algorithms here.